dspace-api/src/main/java/org/dspace/content/RelationshipMetadataValue.java

```diff
@@ -57,4 +59,8 @@ public boolean equals(Object obj) {
         }
         return super.equals(obj);
     }
+
+    public int getRelationshipId() {
```

Comment:
Please add JavaDocs to describe this method.  It looks like it's just extracting the RelationshipID corresponding to the current RelationshipMetadataValue?

Reply:
The JavaDocs have been created